src/module.c

```diff
@@ -870,6 +870,7 @@ int64_t commandKeySpecsFlagsFromString(const char *s) {
         else if (!strcasecmp(t,"delete")) flags |= CMD_KEY_DELETE;
         else if (!strcasecmp(t,"channel")) flags |= CMD_KEY_CHANNEL;
         else if (!strcasecmp(t,"incomplete")) flags |= CMD_KEY_INCOMPLETE;
+        else if (!strcasecmp(t,"variable")) flags |= CMD_KEY_VARIABLE_FLAGS;
```

Comment:
Madelyn argued that "variable" is not clear (if the positions are variable or the flags).,
i'm not comfortable with the long name, but let's take that anyway (same for server,c)